Formula/wtfutil.rb

```diff
@@ -0,0 +1,26 @@
+class Wtfutil < Formula
+  desc "The personal information dashboard for your terminal"
+  homepage "https://wtfutil.com"
+  url "https://github.com/wtfutil/wtf/archive/v0.10.2.tar.gz"
+  sha256 "7ed8c63bd7126b39b35026bf25df3ebd4404aa804c54eb47b6fa04c6fe1e478b"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
+
+    dir = buildpath/"src/github.com/wtfutil/wtf"
+    dir.install buildpath.children
+
+    cd dir do
+      system "go", "mod", "vendor"
+      system "go", "build", "-o", bin/"wtf"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/wtf", "-p", "brew"
```

Comment:
@chenrui333 333 Could we do anything to check for the existence of log files and check them for a string that should be present indicating correct behavior? Or stdout/stderr? This test is better than `-v` but I worry that the return code may be 0 even if problems exist.

Reply:
Sure, I will do one by one.